cinema_service/permissions.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+SAFE_LIST = (
+    "list",
+    "create",
+)
```

Comment:
why do u need this ?

Reply:
Yup that was just testing on how can we redefine permissions, totally redundant )